command/certificate/verify.go

```diff
@@ -164,6 +178,35 @@ func verifyAction(ctx *cli.Context) error {
 		}
 	}
 
+	if validity {
+
+		remainingValidiy := time.Until(cert.NotAfter)
+		totalValidity := cert.NotAfter.Sub(cert.NotBefore)
+
+		percentUsed := ((totalValidity.Hours() - remainingValidity.Hours()) / totalValidity.Hours()) * 100
+
+		red := "\033[31m"
+		green := "\033[32m"
+		yellow := "\033[33m"
+		reset := "\033[0m"
+		
+		if percentUsed >= 100 {
+			fmt.Println(red, "This certificate has already expired.", reset)
+		} else if percentUsed > 90 {
+			fmt.Printf(red,"Leaf is", int(percentUsed), "% through its lifetime.", reset)
```

Comment:
When you use `fmt.Printf` you can do things like: `fmt.Printf("%sLeaf is %d% through it's lifetime.", red, percentUsed)`. The variables  at the end replace the `%s` and `%d`.